Potential bug: Failing test for test_transformed_emulator_grad where model is RadialBasisFunctions

### Description

I've temporarily disabled this test to get #603 merged but we should look into this:

```
FAILED tests/experimental/test_experimental_transformed.py::test_transformed_emulator_grad[RadialBasisFunctions-None-None] - ValueError: Gradient calculation is not supported.
FAILED tests/experimental/test_experimental_transformed.py::test_transformed_emulator_grad[RadialBasisFunctions-None-y_transforms9] - ValueError: Gradient calculation is not supported.
FAILED tests/experimental/test_experimental_transformed.py::test_transformed_emulator_grad[RadialBasisFunctions-x_transforms10-None] - ValueError: Gradient calculation is not supported.
FAILED tests/experimental/test_experimental_transformed.py::test_transformed_emulator_grad[RadialBasisFunctions-x_transforms11-y_transforms11] - ValueError: Gradient calculation is not supported.
```

### Reproduction steps

```bash
`pytest tests/experimental/test_experimental_transformed.py`
```

### Version

main

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### OS

_No response_

--- a/autoemulate/experimental/emulators/radial_basis_functions.py
+++ b/autoemulate/experimental/emulators/radial_basis_functions.py
@@ -14,6 +14,8 @@ class RadialBasisFunctions(PyTorchBackend):
     Wraps the Radial Basis Function Interpolation in PyTorch.
     """
 
+    supports_grad = False
+
     def __init__(  # noqa: PLR0913
         self,
         x: TensorLike,  # noqa: ARG002
@@ -32,7 +34,6 @@ class RadialBasisFunctions(PyTorchBackend):
         self.epsilon = epsilon
         self.degree = degree
         self.device = device
-        self.supports_grad = False
 
     def _fit(self, x: TensorLike, y: TensorLike):
         self.model = RBFInterpolator(

--- a/tests/experimental/test_experimental_transformed.py
+++ b/tests/experimental/test_experimental_transformed.py
@@ -2,6 +2,7 @@ import itertools
 
 import pytest
 import torch
+
 from autoemulate.experimental.emulators import ALL_EMULATORS, GaussianProcessExact
 from autoemulate.experimental.emulators.base import ProbabilisticEmulator
 from autoemulate.experimental.emulators.transformed.base import TransformedEmulator
@@ -64,11 +65,7 @@ def test_transformed_emulator(
 @pytest.mark.parametrize(
     ("model", "x_transforms", "y_transforms"),
     itertools.product(
-        [
-            emulator
-            for emulator in ALL_EMULATORS
-            if emulator.supports_grad and emulator.supports_grad
-        ],
+        [emulator for emulator in ALL_EMULATORS if emulator.supports_grad],
         [
             None,
             [StandardizeTransform()],

--- a/tests/experimental/test_experimental_transformed.py
+++ b/tests/experimental/test_experimental_transformed.py
@@ -2,11 +2,7 @@ import itertools
 
 import pytest
 import torch
-from autoemulate.experimental.emulators import (
-    ALL_EMULATORS,
-    GaussianProcessExact,
-    RadialBasisFunctions,
-)
+from autoemulate.experimental.emulators import ALL_EMULATORS, GaussianProcessExact
 from autoemulate.experimental.emulators.base import ProbabilisticEmulator
 from autoemulate.experimental.emulators.transformed.base import TransformedEmulator
 from autoemulate.experimental.transforms import (
@@ -88,9 +84,6 @@ def test_transformed_emulator_grad(
 ):
     x, y = sample_data_y2d
     x2, _ = new_data_y2d
-    # TODO: investigate why RadialBasisFunctions fails here, issue: #615
-    if model is RadialBasisFunctions:
-        pytest.skip("Gradient failing for RadialBasisFunctions")
     em = TransformedEmulator(
         x, y, x_transforms=x_transforms, y_transforms=y_transforms, model=model
     )

